docs/content/v0.20/examples.md

```diff
@@ -17,6 +17,8 @@ The following steps establish an initial state used in the examples below.
    kflex init --create-kind
    ```
 
+  If you are installing KubeStellar on an existing OpenShift cluster, just use the command `kflex init`.
```

Comment:
Is this true only for OpenShift or for any "existing cluster" ?

Reply:
About the remark about how to cope with OpenShift:

(A) Belongs in a different place than a report on a past successful test.

(B) I am OK with just having a binary distinction between ordinary Kubernetes clusters and OpenShift clusters. OpenShift is by far the leading example of clusters that defy expectations, and we will not soon have enough experience with any others.